### PR TITLE
completed const-let

### PR DESCRIPTION
**Pull request by yadanur**
image

### CODE REVIEW

File: es6-const-let/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      while (i < greetingList.length) {
+        console.log('Current greeting:', greetingList[i]);
+        i++;
+      }
+    }
+    logAllGreetings(greetings);
+
+    function getRandomGreeting(greetingList) {
+      var randomIndex = Math.floor(Math.random() * greetingList.length);
+      var greeting = greetingList[randomIndex];
```
Comment (TimelikeClosure) [inline es6-const-let/index.html]: These should also be changed to `const` or `let`.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: es6-const-let/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      let i = 0;
+      while (i < greetingList.length) {
+        console.log('Current greeting:', greetingList[i]);
+        i++;
+      }
+    }
+    logAllGreetings(greetings);
+
+    function getRandomGreeting(greetingList) {
+      let randomIndex = Math.floor(Math.random() * greetingList.length);
```
Comment (TimelikeClosure) [inline es6-const-let/index.html]: This does not get re-assigned within a single run of this scope, so it should be a constant.

---

Screenshot_2

File: es6-const-let/index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    }
+    console.log('Random greeting:', getRandomGreeting(greetings));
+
+    function addToGreetings(newGreeting, greetingList) {
+      greetingList.push(newGreeting);
+    }
+    addToGreetings('namaste', greetings);
+    logAllGreetings(greetings);
+
+    let variable = 'meow';
```
Comment (TimelikeClosure) [inline es6-const-let/index.html]: Same as above.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: es6-const-let/index.html (hunk 3 above)
Comment (yadanur) [reply es6-const-let/index.html]: Done!

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**